docs/storybook/stories/ui/Checkbox.stories.tsx

```diff
@@ -13,6 +13,13 @@ const Template: Story<CheckboxProps> = (args) => {
 
 export const Example = Template.bind({});
 
+export const ExampleIndeterminated = Template.bind({});
```

Comment:
This example is not working http://docsstorybook-feature-checkbox-indete-staticbucket-cgegxj0la8sg.s3-website-us-east-1.amazonaws.com/?path=/story/ui-checkbox--example-indeterminated

Reply:
I left this defined but i removed and set indeterminate  from true